vit/config/config.sample.ini

```diff
@@ -57,6 +57,9 @@
 #included in the active filter for this setting to have effect.
 #focus_on_add = False
 
+# Boolean. If true, VIT will reload if the tasks where changed on disk.
```

Comment:
I think it might be beneficial to be a bit more descriptive here, since this is the only documentation we're adding at the moment.

First proposal:

Boolean. If true, VIT will monitor TaskWarrior's pending.data file, and reload the currently displayed report if the file is changed.

Reply:
Added, plus some extra information. Thanks!